examples.md

```diff
@@ -437,13 +437,68 @@ When dependencies are installed later in the workflow, we must specify the same
 
 ## Scala - SBT
 
+SBT 1.2
 ```yaml
+- name: Cache SBT ivy cache
+  uses: actions/cache@v2
+  with:
+    path: ~/.ivy2/cache
+    key: ${{ runner.os }}-sbt-ivy-cache-${{ hashFiles('**/build.sbt') }}
```

Comment:
I guess we cannot use the hash of build.sbt, since it's not always changed when dependencies are changed.

No need to see a specific file (like Gemfile.lock, package-lock.json, etc) to know this, downloaded modules are shared globally on a machine (as you can see on the above `~/.ivy2/cache`).

I suppose `restore-keys` can solve this instead, which can found existing cache (on every run if dependecies are changed or not):
 
```yaml
        key: ${{ runner.os }}-sbt-ivy-cache-${{ github.run_id }}
        restore-keys: |
          ${{ runner.os }}-sbt-ivy-cache-
```

(but not sure if this restore-keys can restore last key actually.)

Reply:
Oh right, I forgot you can define dependencies outside `build.sbt`.

Maybe we should always load from the cache regardless if the dependencies have changed?
```yml
  uses: actions/cache@v2
  with:
    path: ~/.ivy2/cache
    key: ${{ runner.os }}-sbt-ivy-cache
```

Reply:
I'm afraid it doesn't work.
An existing cache, according to the documents, should not be updated:

https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows

> Once you create a cache, you cannot change the contents of an existing cache but you can create a new cache with a new key.

and I've found the sentense:

> When the action finds a partial match, the most recent cache is restored to the path directory.

Looks good to use `restore-keys`.

Reply:
Thanks, I've updated the PR accordingly